components/controller.ts

```diff
@@ -514,6 +514,12 @@ export class Controller {
                     lastServerSideData.contracts,
                 )) {
                     this.contracts.set(contractId, contractData)
+
+                    if (contractData.SMF.destinations?.addToDestinations) {
+                        controller.missionsInLocations[
+                            contractData.Metadata.Location
+                        ].push(contractData.Metadata.Id)
+                    }
```

Comment:
Why not use `addBefore` and `addAfter` here? Seems it would be useful.

Reply:
Amogus sus